test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -120,12 +120,13 @@ public HubConnectionTests()
 
         [Theory]
         [MemberData(nameof(HubProtocols))]
+        //TODO: should be across transports
```

Comment:
Yes. This PR is only a very hacky prototype. I am using this test to verify things work end-to-end.